source/initial_conditions/adiabatic_boundary.cc

```diff
@@ -179,16 +138,18 @@ namespace aspect
           prm.declare_entry ("Isotherm depth filename",
                              "adiabatic_boundary.txt",
                              Patterns::FileName(),
-                             "File from which the base of the lithosphere depth data is read. "
```

Comment:
I think this space at the end of the line is needed. You *always* need a space if you continue the string on the next line, you *never* use a space if it is the last line of the string.